src/app/inbound/mod.rs

```diff
@@ -25,19 +24,21 @@ mod set_remote_ip_on_req;
 
 pub use self::endpoint::{Endpoint, RecognizeEndpoint};
 
-pub fn server<P>(
+pub fn spawn<G, P>(
```

Comment:
It's not super obvious to me why this was changed to `spawn` rather than returning a task as it did previously — I'm not necessarily opposed but I'm curious about the rationale.

Reply:
I'm angling towards splitting inbound and outbound into subcrates so they can be compiled in parallel... I initially had this function returning and `impl Accept<...`, but it seemed better to just pass the listener in with it all and bind them together. I could have it return `impl Future`, but i figure this seals off any possibility of the compiler having to expose the inner stack type to the `main`... It's an experiment, I guess.

Reply:
Okay, yeah, that makes sense! Just wondering what the motivation was.